index.d.ts

```diff
@@ -245,3 +250,29 @@ declare const captureWebsite: {
 };
 
 export = captureWebsite;
+interface ClipOptions {
```

Comment:
This should be inside `declare namespace captureWebsite`. And should be named:
```suggestion
interface BoundingBox {
```

Like in Puppeteer.